sig-network/annual-report-2021.md

```diff
@@ -0,0 +1,36 @@
+# 2021 SIG network annual report
+
+This report reflects back on CY 2020 and was written in April 2021.
+
+> Note: Each response should have supporting documentation, linked KEPs, graphs, data
+> Note: Err on the side of being descriptive with your responses. End users and others that read this might not know our upstream acronyms, shorthand, or where to find a reference. 
+
+# Operational
+
+- How are you doing with operational tasks in [sig-governance.md](https://github.com/kubernetes/community/blob/master/committee-steering/governance/sig-governance.md)?
+  - Is your [README](./README.md) accurate? 
```

Comment:
Yes, README looks accurate.

Reply:
copying here for collab https://hackmd.io/SITAZvJ8RLWJy1K5FFmiIQ

Reply:
Let's please keep comments on the PR. I don't want to have to collect comments from a bunch of different locations. I also don't want to have to make a new account to use that site.